query-engine/prisma-models/src/builders/index_builder.rs

```diff
@@ -10,13 +10,20 @@ pub struct IndexBuilder {
 
 impl IndexBuilder {
     pub fn build(self, fields: &[ScalarFieldRef]) -> Index {
+        let name = if self.name.is_none() {
```

Comment:
I've defaulted to always creating a name for an index. I'm not really sure why we did not do this previously. I will submit another PR later to clean up the code everywhere else where we generate the index name